net/miniupnpd/Makefile

```diff
@@ -8,12 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpd
-PKG_VERSION:=2.2.3
+PKG_VERSION:=3.0.1
```

Comment:
Where did you get this version from?

Reply:
That was what I used for testing, forgot to change this.
May I propose 2.3.0-1? Since 2.3.0 is the latest tag.

Reply:
We usually don't use PKG_VERSION when we build packages from a git hash. I'd just drop it.

Reply:
Done!